### DISCUSSION TITLE
Support use of NLB for Aurora Postgres

### DISCUSSION THREAD

**Discussion by vtejapavanreddy** [aws/aws-advanced-jdbc-wrapper#568] [Ideas]
Hi All,

Currently IAM token is being generated based on the host provided. But when NLB is used we have to specify NLB to connect to DB.
Please enhance such that we can provide both NLB & hostname so that token is generated using host name but connection should be made using NLB.

**Comment by crystall-bitquill** [comment]
Hi @vtejapavanreddy, you can specify which host to use to generate the IAM token with [the `iamHost` property](https://github.com/awslabs/aws-advanced-jdbc-wrapper/blob/main/docs/using-the-jdbc-driver/using-plugins/UsingTheIamAuthenticationPlugin.md#how-do-i-use-iam-with-the-aws-advanced-jdbc-driver).

Here is a simple example:
```java
final Properties properties = new Properties();
properties.setProperty("user", "iam_user");
properties.setProperty("wrapperPlugins", "iam");
properties.setProperty("iamHost", "hostname");
properties.setProperty("iamRegion", "region");

String connString = "jdbc:aws-wrapper:postgresql://nlb-url/db";

try (Connection conn = DriverManager.getConnection(connString, properties)) {
    /// ...
}
```
